client/src/components/modules/CommentsBlock.js

```diff
@@ -34,7 +34,7 @@ class CommentsBlock extends Component {
               content={comment.content}
             />
           ))}
-          <NewComment storyId={this.props.story._id} />
+          {this.props.userId && <NewComment storyId={this.props.story._id} />}
```

Comment:
are we teaching this syntax? if not, how should we express this?

Reply:
I think this is a nice thing to teach. It would first be covered in @a-lchen's lecture (I think?) so it's up to him, but I would be in favor of it. Alternatively, it could also be covered for the first time in the auth workshop, but that might be a little too much there? IDK how time crunched that lecture is.

If not, we'd probably need to define some variable and have some logic beforehand for the NewComment, or extract out the NewComment rendering to a function and have the logic there.

Reply:
I agree I think this would be a nice thing to teach. If Alex has time/gets back to you, he could cover it, otherwise just cover in auth workshop. I think this can be merged.